.rubocop_todo.yml

```diff
@@ -1,55 +1,40 @@
 # This configuration was generated by
-# `rubocop --auto-gen-config`
```

Comment:
yes there's a lot of stuff we could/should fix. I would like to get the switch to openvox done first. Otherwise the diff is going to be terrible.

Reply:
I at least split it off to its own commit which makes it a bit easier to review.